database/insertionHelpers.js

```diff
@@ -44,7 +43,7 @@ module.exports.addSearchQuery = (obj) => {
   const {
     searchQueryId, timestamp, market, checkIn, checkOut, roomType,
   } = obj;
-  const queryText = `INSERT INTO listings (search_id, market, searched_at, check_in, check_out, room_type) VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') ON CONFLICT (search_id) DO NOTHING`;
+  const queryText = `INSERT INTO search_queries (search_id, market, searched_at, check_in, check_out, room_type) VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') ON CONFLICT (search_id) DO NOTHING`;
```

Comment:
If possible, try to make long queries formatted into multiple lines

Reply:
Fixed it thanks!